include/config/rcxml.h

```diff
@@ -20,6 +20,12 @@ enum window_switcher_field_content {
 	LAB_FIELD_TITLE,
 };
 
+enum view_mode {
```

Comment:
I have working tearing option in my fork so the plan was to make PR for it very soon, unless you want actual tearing protocol before merging it, currently it enables tearing when current focused_view is fullscreen, or always depending on what you set it to.

Reply:
I don't know now much about adaptive-sync and tearing.

Questions:

1. Are they both global settings (rather than view-specific)?
2. Are they per-output settings? In my mind, fullscreen is just an output-state. 
3. Should they be separate settings? For example should we allow adaptive-sync but not tearing or vice-versa?

My guess is that they are global (or possibly per output) and that we want to be able to set them separately in future... but as I said, I haven't got much insight.

I'm happy to support a config option for tearing. Guess the tearing protocol is for clients to tell the compositor that their surfaces can be torn (tear'd) to avoid applying it to everything... which is obviously a lot better, but if there is a use-case to support tearing quickly I don't mind a config option for it.

Reply:
My current implementions of both are currently global option that should apply to all outputs, but i can only test adaptive sync on 1 monitor that supports it.

They are not related to each other directly other than both can affect if you get tearing. I can enable both and not get tearing in games because then adaptive sync takes care of tearing.

I mean i wanted to use same enum with options DISABLED, ENABLED and FULLSCREEN for both of them. And i heard that tearing protocol is only supported by vulkan in games, so for others i would want support to enable it other way.

How about naming it something like output_mode? Or maybe just call it adaptive_sync_mode for now, and rename if we want that when adding tearing option.